changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.84.1
  * Fixed a bug where simultaneous opening and closing a Realm on different threads might result in a NullPointerException (#1646).
+ * Removed update() method from generated proxy class corresponding to un-updatable model (#1660).
```

Comment:
I don't think we need a changelog update for this. It is an internal change, and invisible to the user.